app/decorators/legislation_decorator.rb

```diff
@@ -41,4 +41,15 @@ def document_links
                 title: document.external? ? document.url : nil
     end
   end
+
+  def instrument_links
+    return [] if model.instruments.empty?
```

Comment:
I think you want to check `model.instruments.nil?` here or remove this check completely because `instruments.map` will handle empty array.

Reply:
Removed this if. Take a look please